yasm/riscv64.patch

```diff
@@ -0,0 +1,21 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -10,11 +10,16 @@ license=('BSD' 'GPL2' 'LGPL2.1' 'PerlArtistic')
+ depends=('glibc')
+ checkdepends=('systemd')
+ options=('staticlibs')
+-source=(https://www.tortall.net/projects/yasm/releases/${pkgname}-${pkgver}.tar.gz)
+-sha256sums=('3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f')
++source=(https://www.tortall.net/projects/yasm/releases/${pkgname}-${pkgver}.tar.gz
++        configure.ac.patch)
```

Comment:
```suggestion
+        configure.ac.patch::https://github.com/yasm/yasm/commit/3e74376b5653102a3957f59005969fcdbbe5a89d.patch)
```
You may use patch link like this to avoid storing patch in our repository. :)
BTW, it is better to use a more meaningful filename, like `fix-autotools-2.70-compat.patch`.

Reply:
Done.